src/averageAges.js

```diff
@@ -1,5 +1,8 @@
 'use strict';
 
+const averageAge = people => people.reduce((totalAge, person) => totalAge
```

Comment:
Clever decision to make separate function for that.
However, if you create variable for the function, its name should start with verb (action).
```suggestion
const getAverageAge = people => people.reduce((totalAge, person) => totalAge
```